templates/pg_hba.conf.j2

```diff
@@ -0,0 +1,11 @@
+#########################################################################################
+# [ WARNING ]
+# postgresql client authentication configuration file maintained by the postgres-operator
+# local changes may be overwritten.
+#########################################################################################
+{%- for connection in connections %}
+{%- if connection.comment is defined %}
+# {{connection.comment}}
```

Comment:
nit: If a developer includes a newline in the comment this will break. I suppose its dev-mode and should be caught, but it might cause unintended confusion later.

Reply:
I agree.
This template will be removed as said in the comment above  (there will be no future changes to the configuration file).